AppCenterDataStorage/AppCenterDataStorage/MSDocumentWrapper.m

```diff
@@ -21,6 +21,13 @@ - (instancetype)initWithDeserializedValue:(id<MSSerializableDocument>)deserializ
                                      eTag:(NSString *)eTag
                           lastUpdatedDate:(NSDate *)lastUpdatedDate {
   if ((self = [super init])) {
+
+    // Create json string.
+    NSError *error;
+    NSData *jsonData = [NSJSONSerialization dataWithJSONObject:[deserializedValue serializeToDictionary] options:0 error:&error];
+    if (!error) {
```

Comment:
I would assume that we'd want at least a warning here if we can't serialize the value